docs/bots-guide.md

```diff
@@ -76,12 +76,18 @@ You need:
 **Note: Please be considerate when testing experimental bots on
   public servers such as chat.zulip.org.**
 
-1. Install all requirements. You can either
-
-    * run `pip install zulip_bots` for a stable version, or
-    * clone the [`zulip_bots`](https://github.com/zulip/python-zulip-api/tree/master/zulip_bots)
-      repository for the latest code. Install it with
-      `pip -e <path/to/zulip_bots>`; you will be able to make changes to the code.
+1. Install all requirements. You can **either**
+
+    * run `pip install zulip_bots` for a stable version, **or**
+    * install the latest version from our repository:
+      * `git clone https://github.com/zulip/python-zulip-api.git` - clone the [python-zulip-api](
+        https://github.com/zulip/python-zulip-api) repository.
+      * `cd python-zulip-api` - navigate into your cloned repository.
+      * `./tools/provision` - install all requirements in a Python virtualenv.
+      * run the `source <activation/path>` command printed in the previous step to activate the virtualenv.
+      * *Finished*. You should now see the name of your venv preceding your prompt, e.g. `(ZULIP-~1)`.</br>
+         *Hint: `./tools/provision` installs `zulip`, `zulip_botserver`, and `zulip_botserver` in developer
```

Comment:
should be `zulip_bots` in the middle here?

Reply:
:( yes.